### ISSUE TITLE
Email bodies should be sub parts

### ISSUE THREAD

**Issue by willpower232** [postalserver/postal#375]
The HTML and plain text parts of an email should be sub parts of a part with content type "multipart/alternative" and the main message content type should be "multipart/mixed". Attachments should be siblings of the "multipart/alternative" and not on the same level as the HTML or plain text parts.

```
"multipart/mixed"
    |
    |\ "multipart/alternative"
    |   |\ "text/plain"
    |   \ "text/html"
    |
    \ attachments
```

This problem presents itself when some recipients say they are not able to see the attachments to the message.

You can also replicate this issue by downloading a sent message within the Postal UI. If you open the EML file in Outlook 2016, you will not be able to see the attachment.

Ruby reference: https://github.com/mikel/mail/issues/522#issue-11549826
Comparable Stack Overflow question: https://stackoverflow.com/questions/3902455/mail-multipart-alternative-vs-multipart-mixed

I will fix this as a pull request shortly...

**Comment by alexander-rieder** [comment]
when will this get merged?

**Comment by willpower232** [comment]
:tada: https://github.com/postalserver/postal/pull/376#event-8731747240

**Comment by catphish** [comment]
Resolved by d0db1345a2bf8f538b01b974e74391da6fffe2b1
